control-plane/stor-port/src/types/v0/store/node.rs

```diff
@@ -343,6 +349,50 @@ impl NodeSpec {
             None => false,
         }
     }
+
+    /// Add the draining volume to the node spec for checking shutdown nexuses.
+    pub fn add_draining_volume(&mut self, volume_id: &VolumeId) {
+        self.draining_volumes.push(volume_id.clone());
+        if self.draining_timestamp.is_none() {
+            self.draining_timestamp = Some(SystemTime::now())
+        }
+    }
+
+    /// Get the vector of draining volumes on this node.
+    pub fn get_draining_volumes(&self) -> Vec<VolumeId> {
```

Comment:
```suggestion
    pub fn draining_volumes(&self) -> Vec<VolumeId> {
```
and below on other get_**

Reply:
fixed